app/src/test/java/org/pesmypetcare/mypetcare/controllers/exercise/TestTrDeleteExercise.java

```diff
@@ -0,0 +1,100 @@
+package org.pesmypetcare.mypetcare.controllers.exercise;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.httptools.utilities.DateTime;
+import org.pesmypetcare.mypetcare.features.pets.Pet;
+import org.pesmypetcare.mypetcare.features.pets.PetRepeatException;
+import org.pesmypetcare.mypetcare.features.pets.events.exercise.Exercise;
+import org.pesmypetcare.mypetcare.features.pets.events.exercise.NotExistingExerciseException;
+import org.pesmypetcare.mypetcare.features.users.NotPetOwnerException;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubPetManagerService;
+import org.pesmypetcare.usermanager.datacontainers.pet.GenderType;
+
+import java.util.concurrent.ExecutionException;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Albert Pinto
+ */
+public class TestTrDeleteExercise {
+    private User user;
+    private Pet pet;
+    private TrDeleteExercise trDeleteExercise;
+
+    @Before
+    public void setUp() throws PetRepeatException {
+        user = new User("johnDoe", "johndoe@gmail.com", "1234");
+        pet = getDinkyPet();
+        trDeleteExercise = new TrDeleteExercise(new StubPetManagerService());
+
+        pet.addExercise(new Exercise("Frisbee", "Playing at the beach",
+            DateTime.Builder.buildFullString("2020-05-04T10:00:00"),
+            DateTime.Builder.buildFullString("2020-05-04T11:00:00")));
+        user.addPet(pet);
+    }
+
+    @Test(expected = NotPetOwnerException.class)
+    public void shouldNotDeleteExerciseToNonOwnedPet() throws NotPetOwnerException, NotExistingExerciseException, ExecutionException, InterruptedException {
+        trDeleteExercise.setUser(new User("johnSmith", "johnsmith@gmail.com", "5678"));
+        trDeleteExercise.setPet(pet);
+        trDeleteExercise.setExerciseDateTime(DateTime.Builder.buildFullString("2020-05-04T10:00:00"));
+        trDeleteExercise.execute();
+    }
+
+    @Test(expected = NotExistingExerciseException.class)
+    public void shouldNotDeleteNonExistingExercise() throws NotPetOwnerException, NotExistingExerciseException, ExecutionException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 154).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)